src/api/spec/models/notification_spec.rb

```diff
@@ -33,4 +33,43 @@
 
     it { expect(test_group.notifications).to include(notification) }
   end
+
+  describe '#user_active?' do
+    subject { rss_notification.user_active? }
+
+    let(:rss_notification) { create(:rss_notification, subscriber: test_user) }
```

Comment:
Nitpick: We usually put the `let` lines before the `subject` line. It helps to know what variables you have declared and what you can use later. It also happens in the next describe section: '#any_user_in_group_active?'.

Reply:
done